### ISSUE TITLE
[Bug] Update from 1.2.1 to 1.2.3 hangs electron app

### ISSUE THREAD

**Issue by luchillo17** [node-schedule/node-schedule#362]
So i have an app made in electron running fine with version `1.2.1`, i update to `1.2.3` (with yarn but i don't think it matters), it freezes and shows this in lower bar:
![image](https://user-images.githubusercontent.com/9698639/27095002-bae1f29c-5031-11e7-87b0-dbad9ec75612.png)

After some time it shows in Mac's activity monitor it shows like this:
![image](https://user-images.githubusercontent.com/9698639/27095025-cdbace02-5031-11e7-904f-6ce2a52b6661.png)

**Comment by santigimeno** [comment]
It's quite possible https://github.com/node-schedule/node-schedule/commit/519917bc8138ebc9a8ad19641a02fbaec89ff69d is the culprit. Can you confirm that?

Anyway we would need more info on the issue, can you provide some code that reproduces it?

**Comment by luchillo17** [comment]
Yeah, i was looking to the deps that get installed when i update and when i saw `cron-parser` i though the same.

Some code? unlikely, that would require to create an electron app, but there's hope, my app is open source, steps:
- Clone develop branch https://github.com/luchillo17/Electron-Webpack/tree/develop
- Install with yarn (I'm using yarn, i say install with yarn to make use of the `yarn.lock` file so you get the exact same dependency tree).
- Use following command to install & run app:
```sh
yarn add node-schedule@1.2.2 && yarn server:dev:hmr
```

Idk if it will happen exactly the same if done with npm (in theory it should).
